lib/MySQL_HostGroups_Manager.cpp

```diff
@@ -3151,6 +3286,195 @@ SQLite3_result * MySQL_HostGroups_Manager::SQL3_Free_Connections() {
 	return result;
 }
 
+void MySQL_HostGroups_Manager::p_update_connection_pool() {
+	for (int i = 0; i < static_cast<int>(MyHostGroups->len); i++) {
+		MyHGC *myhgc { static_cast<MyHGC*>(MyHostGroups->index(i)) };
+		for (int j = 0; j < static_cast<int>(myhgc->mysrvs->cnt()); j++) {
+			MySrvC *mysrvc { static_cast<MySrvC*>(myhgc->mysrvs->servers->index(j)) };
+			std::string endpoint_addr { mysrvc->address };
+			std::string endpoint_port { std::to_string(mysrvc->port) };
+			std::string hostgroup_id { std::to_string(myhgc->hid) };
+			std::string endpoint_id { endpoint_addr + ":" + endpoint_port + ":" + hostgroup_id };
```

Comment:
Personal preference, since the PK is hostgroup+address+port, we should probably keep that order.

Reply:
Changed!